optimizely/event/factory.go

```diff
@@ -67,15 +67,15 @@ func CreateImpressionUserEvent(context EventContext, experiment entities.Experim
 	userEvent := UserEvent{}
 	userEvent.Timestamp = makeTimestamp()
 	userEvent.VisitorID = userContext.ID
-	userEvent.UUID = guuid.New().String()
+	userEvent.Uuid = guuid.New().String()
```

Comment:
nit: This is a little inconsistent with the `EventBatch.UUID`. I think the Go linter complains about `Uuid` having to be `UUID` since it is an acronym